src/features/components/templates/navigation/template/index.tsx

```diff
@@ -68,60 +66,56 @@ const NavTemplate = ({
 
     return (
         <>
```

Comment:
do we need this fragment? 
if not, please remove it, thanks :pray:

Reply:
Done @mohsen-deriv  🫡